assets/src/edit-story/components/panels/textStyle/color.js

```diff
@@ -130,6 +131,11 @@ function ColorControls({ selectedElements, pushUpdate }) {
             icon={<Icon />}
             value={backgroundTextMode === mode}
             label={label}
+            aria-label={sprintf(
+              /* translators: %s: Text background mode. */
+              __('Set text background mode: %s', 'web-stories'),
+              mode
```

Comment:
Maybe this should be `label` rather than `mode`? It's basically the same, but `label` is less shouty.

Reply:
Ah yes, as for the overlay mode, `label` is localizable, `mode` is not.